scan/lib/scan/detect_values.rb

```diff
@@ -37,20 +37,30 @@ def self.filter_simulators(simulators, deployment_target)
     end
 
     def self.default_device_ios
-      config = Scan.config
-
-      if config[:device] # make sure it actually exists
-        device = config[:device].to_s.strip.tr('()', '') # Remove parenthesis
-
-        found = FastlaneCore::Simulator.all.find do |d|
-          (d.name + " " + d.ios_version).include? device
+      devices = Scan.config[:devices] || Array(Scan.config[:device]) # important to use Array(nil) for when the value is nil
+      found_devices = []
+
+      if (devices || []).count > 0
```

Comment:
Is it possible for `devices` to be `nil` here? `Array(nil)` equals `[]` so I would think that the last possible value devices could be initialized to is already `[]`?